### PR TITLE
Add option in preferences to show/hide status bar

### PR DESCRIPTION
**Pull request by goanpeca**
Related to #2353 

## Description
Add option in preferences to show/hide status bar

**On**
image

**Off**
image

### CODE REVIEW

File: spyderlib/spyder.py [hunk 1]
```diff
@@ -1270,6 +1270,9 @@ def post_visible_setup(self):
         if not self.light:
             # Update lock status of dockidgets (panes)
             self.lock_dockwidgets_action.setChecked(self.dockwidgets_locked)
+
+            # Update lock status of dockidgets (panes)
+            self.lock_dockwidgets_action.setChecked(self.dockwidgets_locked)
```
Comment (ccordoba12) [inline spyderlib/spyder.py]: This block is repeated, please remove it :-)

Comment (goanpeca) [inline spyderlib/spyder.py]: Whoops, accident

**Comment by goanpeca** [comment]
@ccordoba12 fixed!

**Comment by ccordoba12** [comment]
Thanks! This is working as expected :-)

**Comment by goanpeca** [comment]
:+1:

**Pull request merged**